packages/sockethub/src/bootstrap/platforms.js

```diff
@@ -66,28 +67,23 @@ module.exports = function platformLoad(moduleList) {
         const packageJson = require(path + '/package.json');
         let types = [];
 
+        const validate_platform_schema = ajv.compile(schemas.platform);
         // validate schema property
-        if (! tv4.validate(p.schema, schemas.platform)) {
+        if (! validate_platform_schema(p.schema)) {
           throw new Error(
-            `${platformName} platform schema failed to validate: ${tv4.error.message}`);
+            `${platformName} platform schema failed to validate: `, ajv.errors);
```

Comment:
Can you just add the `ajv.errors` as second argument when creating a new `Error` object?

If this is the standard JS `Error`, then the second argument needs to be an object with a `cause` property: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error